docs/OpenFgaApi.md

```diff
@@ -2301,6 +2303,167 @@ No authorization required
+            System.err.println("Reason: " + e.getResponseBody());
+            System.err.println("Response headers: " + e.getResponseHeaders());
+            e.printStackTrace();
+        }
+    }
+}
+```
+
+### Parameters
+
+
+| Name | Type | Description  | Notes |
+|------------- | ------------- | ------------- | -------------|
+| **storeId** | **String**|  | |
+| **body** | [**ListObjectsRequest**](ListObjectsRequest.md)|  | |
+
+### Return type
+
+CompletableFuture<[**StreamResultOfStreamedListObjectsResponse**](StreamResultOfStreamedListObjectsResponse.md)>
+
+
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+- **Content-Type**: application/json
+- **Accept**: application/json
+
+### HTTP response details
+| Status code | Description | Response headers |
+|-------------|-------------|------------------|
+| **200** | A successful response.(streaming responses) |  -  |
+| **400** | Request failed due to invalid input. |  -  |
+| **401** | Not authenticated. |  -  |
+| **403** | Forbidden. |  -  |
+| **404** | Request failed due to incorrect path. |  -  |
+| **409** | Request was aborted due a transaction conflict. |  -  |
+| **422** | Request timed out due to excessive request throttling. |  -  |
+| **500** | Request failed due to internal server error. |  -  |
+
+## streamedListObjectsWithHttpInfo
+
+> CompletableFuture<ApiResponse<StreamResultOfStreamedListObjectsResponse>> streamedListObjects streamedListObjectsWithHttpInfo(storeId, body)
+
+Stream all objects of the given type that the user has a relation with
+
+The Streamed ListObjects API is very similar to the the ListObjects API, with two differences:  1. Instead of collecting all objects before returning a response, it streams them to the client as they are collected.  2. The number of results returned is only limited by the execution timeout specified in the flag OPENFGA_LIST_OBJECTS_DEADLINE.  
+
+### Example
+
+```java
+// Import classes:
+import dev.openfga.sdk.api.client.ApiClient;
+import dev.openfga.sdk.api.client.ApiException;
+import dev.openfga.sdk.api.client.ApiResponse;
+import dev.openfga.sdk.api.configuration.Configuration;
+import dev.openfga.sdk.api.client.models.*;
+import dev.openfga.sdk.api.OpenFgaApi;
+import java.util.concurrent.CompletableFuture;
+
+public class Example {
+    public static void main(String[] args) {
+        ApiClient defaultClient = Configuration.getDefaultApiClient();
+        defaultClient.setBasePath("http://localhost");
+
+        OpenFgaApi apiInstance = new OpenFgaApi(defaultClient);
+        String storeId = "storeId_example"; // String | 
+        ListObjectsRequest body = new ListObjectsRequest(); // ListObjectsRequest | 
+        try {
+            CompletableFuture<ApiResponse<StreamResultOfStreamedListObjectsResponse>> response = apiInstance.streamedListObjectsWithHttpInfo(storeId, body);
+            System.out.println("Status code: " + response.get().getStatusCode());
```

Comment:
All these printlin's can use a helper function. This references another comment I made regarding the same usage of a helper function.

Reply:
Did you mean using a logger/structured logger
Usually from what I've seen the SDK doesn't use any of those
I guess we can avoid introducing that dependency for it and let the user of the SDK choose what they want to log using
Unless you see some truly repetitive code that can be removed